mcs/tools/nunit-lite/nunit-lite-console/nunit-lite-console.exe.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="Windows-1252"?>
+<configuration>
+ <runtime>
+   <legacyUnhandledExceptionPolicy enabled="1" />

```

Comment:
please add a comment on why we need this, I remember there was some discussion around it.